web/src/components/apps/LicenseTester.jsx

```diff
@@ -0,0 +1,32 @@
+import useLicense from "@features/App/api/getLicense";
+import { useLicenseWithIntercept } from "@features/App/api/useLicense";
+import React, { useEffect } from "react";
+import { usePrevious } from "@src/hooks/usePrevious";
+
+const LicenseTester = ({ appSlug, setLoader }) => {
+  const { licenseQuery, isSlowLoading } = useLicenseWithIntercept(appSlug);
+  // licenseQuery includes everything that react-query returns
+  const { data } = licenseQuery;
+  const { license } = data || {};
+  const { entitlements } = license || [];
+
+  useEffect(() => {
+    setLoader(isSlowLoading);
+  }, [isSlowLoading, licenseQuery?.data]);
+
+  return (
+    <div>
```

Comment:
test component, can be deleted when it seems unnecessary. just wanted to leave an example here

Reply:
thank you